contrib/babelfishpg_tsql/src/pl_exec.c

```diff
@@ -10207,3 +10211,21 @@ bool reset_search_path(PLtsql_stmt_execsql *stmt, char *old_search_path, bool* r
 	}
 	return false;
 }
+
+/*
+ * Set the global variable original_query_string with the original query.
+ */
+static void
+set_original_query_string(char *queryString)
```

Comment:
Is there a specific reason to have this function?

Reply:
It's not needed anymore. Removed it.